Restarting Win J25 CI following

```
[ERROR] Errors: 
[ERROR]   CbusAllocateNodeNumberTest.testDisplayDialogue � JUnit Failed to close extension context
[INFO] 
[ERROR] Tests run: 48184, Failures: 0, Errors: 1, Skipped: 1588
```

--- a/help/en/releasenotes/current-draft-note.shtml
+++ b/help/en/releasenotes/current-draft-note.shtml
@@ -49,8 +49,7 @@
 
         <h4>Direct</h4>
             <ul>
-              <li>Added LokPilot 5 micro DCC Direct Atlas Legacy</li>
-              <li>Renamed LokPilot 5 Micro Next18 DCC to LokPilot 5 micro Next18 DCC (capitalization) to match ESU</li>
+                <li></li>
             </ul>
 
         <h4>ESU</h4>
@@ -300,6 +299,8 @@
         <h4>ESU</h4>
             <ul>
                 <li>LokSound 5 nano DCC now declares its E24 interface.</li>
+                <li>Added LokPilot 5 micro DCC Direct Atlas Legacy.</li>
+                <li>Renamed LokPilot 5 Micro Next18 DCC to LokPilot 5 micro Next18 DCC (capitalization) to match ESU.</li>
                 <li></li>
             </ul>
 

